credsweeper/filters/value_pem_pattern_check.py

```diff
@@ -17,3 +19,4 @@ def __init__(self, config: Config):
         """
         super().__init__(config)
         self.pattern_len = config.pem_pattern_len
+        self.pattern = re.compile(fr"(.)\1{{{str(self.pattern_len - 1)},}}")
```

Comment:
It seems this line is same as `ValuePatternCheck` one...
BTW isn't it useless from `ValuePemPatternCheck`?